forge-gui/res/cardsfolder/upcoming/churning_reservoir.txt

```diff
@@ -0,0 +1,9 @@
+Name:Churning Reservoir
+ManaCost:R
+Types:Artifact
+T:Mode$ Phase | Phase$ Upkeep | ValidPlayer$ You | Execute$ TrigPutCounter | TriggerZones$ Battlefield | TriggerDescription$ At the beginning of your upkeep, put an oil counter on another target nontoken artifact or creature you control.
+SVar:TrigPutCounter:DB$ PutCounter | ValidTgts$ Artifact.nonToken+YouCtrl+Other,Creature.YouCtrl+Other | CounterType$ OIL| CounterNum$ 1
+A:AB$ Token | Cost$ 2 T | TokenScript$ r_1_1_phyrexian_goblin | CheckSVar$ CountCountersRemoved | CheckSVarCompare$ GE1 | SpellDescription$ Create a 1/1 red Phyrexian Goblin creature token. Activate only if an oil counter was removed from a permanent you controlled this turn or a permanent with an oil counter on it was put into a graveyard this turn.
+SVar:CountCountersRemoved:Count$CountersRemovedThisTurn OIL Card.YouCtrl

```

Comment:
Alright, here's a silly thing :D

_Skullbriar_ with oil counter -> cast _Suspend_ on him -> Use _Clockspinning_ to remove oil counter in Exile

Can you activate it?

Reply:
Yes I can, is this how it's supposed to work ? :)

Reply:
No, that's not allowed

you can probably add `+inRealZoneBattlefield` to the Valid for it